### ISSUE TITLE
[BUG] Import is not working from Admin Dashboard

### ISSUE THREAD

**Issue by zeealik** [Baboo7/strapi-plugin-import-export-entries#95]
This is happening at the data import of bulk JSON entries at the import button. I tried the same way a few days ago and it worked, I imported more than 5462 entries already using it, but now it's not working. The error is giving me this:
```
error: Cannot read properties of undefined (reading 'attributes')
TypeError: Cannot read properties of undefined (reading 'attributes')
    at getModelAttributes (/test-project/node_modules/strapi-plugin-import-export-entries/server/utils/models.js:89:46)
    at Object.getModelAttributesEndpoint [as getModelAttributes] (/test-project/node_modules/strapi-plugin-import-export-entries/server/controllers/admin/import-controller/get-model-attributes.js:8:26)
```
**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'Import Export' from Strapi Dashboard
2. Click on 'Import'
3. Add the JSON file you want to upload, and press submit.
4. Check your strapi terminal logs at VSCode (any of your editors)

**Expected behaviour**
It should import data after few seconds when I press submit.

**Comment by zeealik** [comment]
I found an alternative way, and that is to import data from selecting content manager > import.

**Comment by Baboo7** [comment]
@zeealik hey! This is fixed in version 1.19.0
